src/commands/hardis/project/audit/apiversion.ts

```diff
@@ -11,6 +11,8 @@ import { catchMatches, generateReports, uxLog } from '../../../../common/utils/i
 Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
 const messages = Messages.loadMessages('sfdx-hardis', 'org');
 
+export const CONSTANTS = {API_VERSION: process.env.SFDX_API_VERSION || '62.0',}
```

Comment:
Here, plz use the CONSTANTS var from https://github.com/hardisgroupcom/sfdx-hardis/blob/main/src/config/index.ts